controller/src/main/java/io/pravega/controller/store/stream/Stream.java

```diff
@@ -308,9 +308,11 @@
 
     /**
      * Returns the currently active stream epoch.
+     *
+     * @param ignoreCached if ignore cache is set to true then fetch the value from the store. 
      * @return currently active stream epoch.
      */
-    CompletableFuture<Pair<Integer, List<Integer>>> getActiveEpoch();
+    CompletableFuture<Pair<Integer, List<Integer>>> getActiveEpoch(boolean ignoreCached);
```

Comment:
it should we used when caller wants to explicitly make sure that the value is fetched from the store.
its interesting in scenarios where value 'reads' and value 'writes' can potentially happen on different controller instances.. and value 'read' wants to always use the latest value from the store.. hence ignore cached and fetch from the store.